src/main/scala/vectorpipe/osm/Features.scala

```diff
@@ -0,0 +1,22 @@
+package vectorpipe.osm
+
+import geotrellis.vector._
+import org.apache.spark.rdd.RDD
+
+// --- //
+
+/** A meaningful wrapping around the various geometry types that can be produced
+  * from the OSM-to-Features conversion. By keeping each Geometry subtype separate,
+  * the user can discard types they don't need.
+  */
+case class Features(
+  points:     RDD[Feature[Point, ElementMeta]],
+  lines:      RDD[Feature[Line, ElementMeta]],
+  polygons:   RDD[Feature[Polygon, ElementMeta]],
+  multiPolys: RDD[Feature[MultiPolygon, ElementMeta]]
+) {
+  /** Flatten the separated features into a single `RDD` of their supertype, `Geometry`. */
+  def geometries: RDD[OSMFeature] = points.sparkContext.union(
+    points.map(identity), lines.map(identity), polygons.map(identity), multiPolys.map(identity)
```

Comment:
What's the purpose of `.map(identity)`?

Reply:
Spark RDDs are "invariant", you can't willy-nilly put things of different datatypes (or mixed super/subtypes) in them. `.map(identity)` is a trick to cast each of the subtypes up to `Geometry` to get around this.